agent/agent-profiler/docs/README.md

```diff
@@ -1,101 +1,120 @@
+Configuration of the profiler triggering settings, such as thresholds and profiling periods can be
+performed within the ApplicationInsights UI under the Performance, Profiler, Triggers UI as
+described in [Installation](#Installation).
+
+Additionally, a number of parameters can be configured within environment variables and the
+`applicationinsights.json` configuration file.
+
+### Environment variables
 
 - APPLICATIONINSIGHTS_PROFILER_ENABLED: boolean (default false)
-    - Can be set as an environment variable to enable the profiler.
-    - While profiling is in beta it must be actively enabled.
-- APPLICATIONINSIGHTS_PREVIEW_METRIC_INTERVAL_SECONDS: int (default 60)
-    - Can be set as an environment variabl.
-    - Period in seconds that performance counters are sampled. This directly impacts the frequency that
-      Alerting thresholds are evaluated. Alerting thresholds by default apply an average over the last 2 min,
-      at the default of 60 seconds this will be 2 samples.
+  - Enables/Disables the profiling feature.
 
-Profiler may also be enabled via adding a config section to applicationinsights.json:
+### Configuration file
 
-```
+```json
 {
   "preview": {
     "profiler": {
-      "configPollPeriodSeconds": 60,
-      "enabled": true
+      "enabled": true,
+      "configPollPeriodSeconds": 50,
+      "cpuTriggeredSettings": "profile_without_env_data",
+      "memoryTriggeredSettings": "profile_without_env_data"
     }
   }
 }
+
 ```
 
-### Json Configuration Parameters
-`configPollPeriodSeconds` - The profiler periodically polls for configuration changes made within the UI.
-
-
-`memoryTriggeredSettings` - This configuration will be used in the event of a memory profile is requested. This can be one of:
-  - "profile" (default value). Uses the `profile` jfc configuration that ships with JFR.
-    
-  - "profile_without_env_data". Uses a profile similar to the `profile` jfc configuration that ships with JFR, however
-    with the following settings disabled:
-    - jdk.JVMInformation
-    - jdk.InitialSystemProperty
-    - jdk.OSInformation
-    - jdk.InitialEnvironmentVariable
-    - jdk.SystemProcess
-  - A path to a custom jfc configuration file on the file system.
-
-`cpuTriggeredSettings` - This configuration will be used in the event of a cpu profile is requested. This can be one of:
-- "profile" (default value). Uses the `profile` jfc configuration that ships with JFR.
-
-- "profile_without_env_data". Uses a profile similar to the `profile` jfc configuration that ships with JFR, however
-  with the following settings disabled:
-    - jdk.JVMInformation
-    - jdk.InitialSystemProperty
-    - jdk.OSInformation
-    - jdk.InitialEnvironmentVariable
-    - jdk.SystemProcess
-- A path to a custom jfc configuration file on the file system.
+`configPollPeriodSeconds` - The profiler periodically polls for configuration changes made within
+the UI. This can control the frequency of that poll.
```

Comment:
I'm not sure we should make this setting public, maybe just as a `debug` setting?